Tests/UnitTests/Core/Material/AverageMaterialsTests.cpp

```diff
@@ -59,8 +61,9 @@ BOOST_AUTO_TEST_CASE(CombineSlabsVacuum) {
 BOOST_AUTO_TEST_CASE(CombineSlabsPercent) {
   auto slab = combineSlabs(percent, percent);
   // combining two identical slabs must give the same average material
-  BOOST_CHECK(slab.material());
-  BOOST_CHECK_EQUAL(slab.material(), percent);
+  BOOST_CHECK(slab.isValid());
+  BOOST_CHECK(slab.material().isValid());
+  BOOST_CHECK_EQUAL(slab.material(), percent.material());
```

Comment:
Oh boy, this used to do a boolean comparison, didn't it?

Reply:
I think so. Very surprising to me, that almost all tests still worked afterwards.